src/Array.luna

```diff
@@ -16,6 +16,10 @@ class Array a:
         wrapper' = wrapper
         ArrayVal self.typeTag wrapper'
 
+    def shortRep:case self of
+        Array t: "Array"
+        _: "Array<1x" + self.length.toText + ">"
```

Comment:
I'd prefer having the common function in Utils that pretty prints this string from row and column counts — and then just call it with proper numbers from Array, ChunkedArray, Column and Table.
That will reduce duplication and will keep shortRep consistent if we change it later.